pcf8523/pcf8523.go

```diff
@@ -0,0 +1,127 @@
+// Package pcf8523 implements a driver for the PCF8523 CMOS Real-Time Clock (RTC)
+//
+// Datasheet: https://www.nxp.com/docs/en/data-sheet/PCF8523.pdf
+package pcf8523
+
+import (
+	"time"
+	"tinygo.org/x/drivers"
+)
+
+const DefaultAddress = 0x68
```

Comment:
might be best to put all of the register const values into a separate `registers.go` file?